File: tests/rest-assured-java/src/test/java/com/portfolio/qa/clients/HealthClient.java
package com.portfolio.qa.clients;

import com.portfolio.qa.endpoints.Endpoints;
import io.restassured.response.Response;

public class HealthClient extends BaseApiClient {

    public Response getHealth() {
        return request()
            .when()
            .get(Endpoints.HEALTH);
    }
}

File: tests/playwright-ts/src/clients/health-client.ts
import type { APIRequestContext, APIResponse, TestInfo } from '@playwright/test';
import { Endpoints } from '../core/endpoints.js';
import { BaseApiClient } from './base-api-client.js';

export class HealthClient extends BaseApiClient {
  constructor(request: APIRequestContext, testInfo: TestInfo) {
    super(request, testInfo);
  }

  async getHealth(): Promise<APIResponse> {
    await this.attachRequest('Get Health', 'GET', Endpoints.health);

    const response = await this.request.get(Endpoints.health);

    await this.attachResponse('Get Health', response);
    return response;
  }
}